druid/src/widget/container.rs

```diff
@@ -12,54 +12,59 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A convenience widget that combines common styling and positioning widgets.
+//! Draws a border and a background around a widget or a widget tree.
```

Comment:
This leaves out that this widget can have rounded corners. I don't think the previous docstring is too bad in spirit, just in that it incorrectly describes the implementation (it describes a previous implementation). I might just go with "A widget that provides simple visual styling options to a child."? Or something like this?

Reply:
Yeah, the previous docstring looks is very similar to the Flutter container docstring. But the implementation is quite different. I like your suggestion. Short and to the point.
When things are more stable I would like to see much more extensive docstrings for all widgets to be honest. This is an important part of our public API so it should be well documented.